docs/alerts/monitors/create-monitor.md

```diff
@@ -62,6 +62,77 @@ To create a monitor from the [Metrics Explorer](/docs/metrics/metrics-queries/me
 Note that the same threshold translating functionality supports to [Opening Alerts Response Page in the Metrics Explorer](/docs/alerts/monitors/alert-response/#translating-thresholds) and [Opening Monitor in the Metrics Explorer](/docs/alerts/monitors/settings/#view-in-metrics-explorer).
 :::
 
+
+<!-- This will be consolidated and moved to the appropriate parts of the doc after
```

Comment:
```suggestion
<!-- This will be consolidated and moved to the appropriate parts of the doc after https://github.com/SumoLogic/sumologic-documentation/pull/4593
```